perception/detection_by_tracker/src/detection_by_tracker_core.cpp

```diff
@@ -159,6 +159,17 @@ DetectionByTracker::DetectionByTracker(const rclcpp::NodeOptions & node_options)
 
   ignore_unknown_tracker_ = declare_parameter<bool>("ignore_unknown_tracker", true);
 
+  // set max search distance for each object
+  using Label = autoware_auto_perception_msgs::msg::ObjectClassification;
+  max_search_distance_for_merger_[Label::UNKNOWN] = 5.0;
+  max_search_distance_for_merger_[Label::CAR] = 5.0;
+  max_search_distance_for_merger_[Label::TRUCK] = 8.0;
+  max_search_distance_for_merger_[Label::BUS] = 8.0;
+  max_search_distance_for_merger_[Label::TRAILER] = 8.0;
+  max_search_distance_for_merger_[Label::MOTORCYCLE] = 2.0;
+  max_search_distance_for_merger_[Label::BICYCLE] = 1.0;
+  max_search_distance_for_merger_[Label::PEDESTRIAN] = 1.0;
```

Comment:
@yukkysaito 
How do you think about this maximum search distance in each class?

Reply:
I updated these parameters and add distance map for divider in 0214f785ea535580376ab9e448f0abde6c577ba5.